css/style.css

```diff
@@ -0,0 +1,25 @@
+form {
+  background: rgba(27, 27, 50, 0.8);
+  border-radius: 0.25rem;
+  padding: 2.5rem
+;
+}
+
+body {
+  color: white !important;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
```

Comment:
It would be great to use margin-top or padding-top for clear picture of the user.